examples/image_border.py

```diff
@@ -0,0 +1,42 @@
+"""
+Layer Border Overlay
```

Comment:
```suggestion
Layer Bounding Box (Border) Overlay
```
For better searching?